src/nni_manager/core/nniDataStore.ts

```diff
@@ -87,7 +87,19 @@ class NNIDataStore implements DataStore {
         event: TrialJobEvent, trialJobId: string, hyperParameter?: string, jobDetail?: TrialJobDetail): Promise<void> {
         this.log.debug(`storeTrialJobEvent: event: ${event}, data: ${hyperParameter}, jobDetail: ${JSON.stringify(jobDetail)}`);
 
-        return this.db.storeTrialJobEvent(event, trialJobId, hyperParameter, jobDetail).catch(
+        let timestamp: number | undefined;
```

Comment:
could you add comments to explain the logic a little bit?

Reply:
Thanks, updated.